### PR TITLE
Create account message

### PR DESCRIPTION
**Pull request by amiaoh**

### CODE REVIEW

File: app/AccountCreation.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ export function AccountCreation(props: {
             type="text"
             placeholder="enter your username"
             value={username}
-            onChange={(e) => {
+            onChange={(e:any) => {
```
Comment (yungyhan) [inline app/AccountCreation.tsx]: any reason this is now required when previously it wasn't? also we're never allowed to use `any`

File: app/App.tsx [hunk 2]
```diff
@@ -12,16 +12,20 @@ enum PageDisplay {
...
         <Login
           onLoginSuccess={() => {
             setdisplayPage(PageDisplay.Welcome);
+            setDisplayAccountCreatedMessage(false)
           }}
           onAccountCreationClick={() => {
             setdisplayPage(PageDisplay.AccountCreation);
+            setDisplayAccountCreatedMessage(false)
           }}
+onAccountCreationSuccess={displayAccountCreatedMessage===true?"Account created":null}
```
Comment (yungyhan) [inline app/App.tsx]: `onAccountCreationSuccess` sounds like it's expecting a function, not a string.

in the ternary, you have  `displayAccountCreatedMessage===true`, is there any way to simplify this?

**Review by yungyhan — CHANGES_REQUESTED** [review]
with how closely the account creation screen interacts with the login screen, it kinda feels like the account creation should be a child view of login. thoughts?

File: app/AccountCreation.tsx (hunk 1 above)
Comment (amiaoh) [reply app/AccountCreation.tsx]: https://github.com/auberginearms/groceryDatabaseApp/pull/10/commits/af12b81a211058fa794bea8a0be303d6b06a1b60 
I don't know why it flagged last time, but it's been removed and it's not complaining

File: app/App.tsx (hunk 2 above)
Comment (amiaoh) [reply app/App.tsx]: https://github.com/auberginearms/groceryDatabaseApp/pull/10/commits/bc86edd35f30606339369565ad37ae4d78e0d787

**Comment by amiaoh** [comment]
making account creation a child only makes sense to me if what we currently see in 'login' is also a child so that we can share the input fields and only change the headers + buttons. however, it seems unnecessary if we're not planning on adding other types of 'welcome' screen children

**Pull request closed without merging**